localization/react-intl/src/app/components/article/ArticlesSidebar.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "id": "articlesSidebar.success",
+    "description": "Banner displayed after an article is successfully added to an item.",
+    "defaultMessage": "Article added successfully!"
+  },
+  {
+    "id": "articlesSidebar.noArticlesAddedToItem",
+    "description": "Message displayed on articles sidebar when an item has no articles.",
+    "defaultMessage": "No articles are being delivered to Tipline users who send in request with that match this Media."
```

Comment:
I found this sentence a bit weird. How about `...who send requests that match this Media."

Reply:
Defer to @brianfleming - I just copied from the design.

Reply:
Oh yeah, that is a typo for sure. Thats what I get for not proofreading designs. @amoedoamorim is correct, lets go with:

`No articles are being delivered to Tipline users who send requests that match this Media.`

Reply:
Fixed!

Reply:
Noice!